CHANGES.rst

```diff
@@ -8,6 +8,19 @@ details, see the commit logs at http://github.com/scikit-build/scikit-build
 Scikit-build 0.12.0
 ===================
 
+The scikit-build GitHub organization welcomes :user:`henryiii` and :user:`mayeut` as core contributors
+and maintainers.
+
+:user:`henryiii` has been instrumental to add Apple Silicon support, add support for Visual Studio 2019, update
+the Continuous Integration infrastructure as well as helping review & integrate contributions, and address
+miscellaneous issues. Additionally, :user:`henryiii` worked on adding `example project <https://github.com/pybind/scikit_build_example>`_
+to built with ``pybind11`` and ``scikit-build``.
+
+:user:`mayeut` is the maintainer of `cibuildwheel <https://cibuildwheel.readthedocs.io>`_ and
+focused his effort on updating the ``cmake-python-distributions`` and ``ninja-python-distributions`` so
+that the corresponding wheels are available on all supported platforms including Apple Silicon and all flavors
+of manylinux.
+
```

Comment:
```suggestion
:user:`mayeut` is a `manylinux <https://github.com/pypa/manylinux>`_ maintainer and
focused his effort on updating the ``cmake-python-distributions`` and ``ninja-python-distributions`` so
that the corresponding wheels are available on all supported platforms including Apple Silicon and all flavors
of manylinux.

```

@mayeut is out for a couple of weeks, but I think this is a bit better, if you want to list related projects we are maintainers of.